opta/constants.py

```diff
@@ -24,6 +25,12 @@
 DEV_VERSION = "dev"
 
 SESSION_ID = int(time.time() * 1000)
+UPGRADE_WARNINGS: Dict[str, str] = {
```

Comment:
Can you document the format of this constant? Is the key the last version where the message is *not* printed? or is it the first version where we print the message?

Reply:
done